packages/deployment/ansible/roles/cosmos-genesis/files/faucet-helper.sh

```diff
@@ -5,21 +5,18 @@ thisdir=$(dirname -- "$0")
 FAUCET_HOME=$thisdir/../faucet
 
 MAX_LINES=-1
-DELEGATE_COINS=62000000ubld,9300000urun
-SOLO_COINS=250000000000urun
+STAKE=75000000ubld
+# GIFT=251000000000urun
+GIFT=100000000urun
```

Comment:
These changes look familiar... did you make them earlier without checking them in or something?

Reply:
Yeah, they were on the deployment controller, but not in the sdk.